enterprise_gateway/services/processproxies/processproxy.py

```diff
@@ -518,7 +518,27 @@ async def launch_process(self, kernel_cmd: str, **kwargs: dict[str, Any] | None)
 
         self._enforce_authorization(**kwargs)
 
-        self.log.debug("BaseProcessProxy.launch_process() env: {}".format(kwargs.get("env")))
+        # Filter sensitive values from being logged
+        sensitive_env = kwargs.get("sensitive_env")
+        if sensitive_env is None:
+            sensitive_env = [
+                "pwd",
+                "password",
+                "secret",
+                "token",
+                "key",
+                "code",
+                "auth",
+                "state",
+                "xsrf",
+            ]
+
+        env_copy = kwargs.get("env").copy()
+        for key in list(env_copy):
+            if any(phrase in key.lower() for phrase in sensitive_env):
+                env_copy[key] = "[secret]"
```

Comment:
Similarly, we should expose a configurable string named `RedactionMask` (env: `EG_REDACTION_MASK`) which _could_ default to `"[secret]"`, although I think using `********` is probably a common default value for a redaction mask.

Reply:
Yes.  Since we're offering this redaction capability (which I don't necessarily agree with in the first place) I think it would be good to offer a configurable mask value.  Organizations roll up logs and may have a common redaction mask they use to identify redacted areas.

If we want to change the default mask to `"[secret]"`, I'm okay with that, but it should be configurable.